app/assets/stylesheets/variables.scss

```diff
@@ -11,6 +11,7 @@ $floral-white: #f9f6ef;
 $tahuna-sands: #d2c295;
 $tea: #b6b1a9;
 $silver: #c5c5c5;
+$black10: rgba(black, 0.1);
```

Comment:
Do we need this?  Can we use `--bs-gray-100` instead?

Reply:
They're not the same:
![image](https://github.com/user-attachments/assets/c903aed2-83fa-4115-ab70-85c2aaf9bf6c)

It seems that `gray-*` is entirely different than `rgba(black, 0.*)`.

Reply:
I think it's hard to tell what is comparable if you're using an alpha channel. Why not pick a solid color?